app/cmd/doctor.go

```diff
@@ -0,0 +1,208 @@
+package cmd
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/jenkins-zh/jenkins-cli/client"
+	"github.com/spf13/cobra"
+)
+
+// DoctorOption is the doctor cmd option
+type DoctorOption struct {
+	OutputOption
+
+	RoundTripper http.RoundTripper
+}
+
+var doctorOption DoctorOption
+
+func init() {
+	rootCmd.AddCommand(doctorCmd)
+}
+
+var doctorCmd = &cobra.Command{
+	Use:   "doctor",
+	Short: "Check your Jenkins config list status and current Jenkins config plugins",
+	Long:  `Check your Jenkins config list status and current Jenkins config plugins`,
+	Run: func(cmd *cobra.Command, _ []string) {
+		jenkinsNames := getJenkinsNames()
+		outString := ""
+		if outputDuplicateNames, err := checkNameDuplicate(jenkinsNames); err == nil {
+			outString += outputDuplicateNames
+		}
+		jenkinsServers := getConfig().JenkinsServers
+		jclient := &client.PluginManager{
+			JenkinsCore: client.JenkinsCore{
+				RoundTripper: doctorOption.RoundTripper,
+			},
+		}
+		if outputJenkinsStatus, err := checkJenkinsServersStatus(jenkinsServers, jclient); err == nil {
+			outString += outputJenkinsStatus
+		}
+		if outputCurrentPluginStatus, err := checkCurrentPlugins(jclient); err == nil {
+			outString += outputCurrentPluginStatus
+		}
+		outString += "Checked is done.\n"
+		cmd.Print(outString)
```

Comment:
So, you just don't print any contents if the err is not nil? Why? Errors cannot be ignored.